test/extra/test_aiohttp.py

```diff
@@ -4,9 +4,12 @@
 
 from ..apps import _aiohttp
 
+from schemathesis.constants import USER_AGENT
+
 
 def test_exact_port():
     app = _aiohttp.create_openapi_app(("success", "failure"))
     run_server(app, 8999)
-    response = requests.get("http://localhost:8999/schema.yaml")
+    headers = {"User-Agent": USER_AGENT}
```

Comment:
Changes in this test don't seem relevant to the test purpose - it only checks if starting a server on some specific port works. Adding headers to this request is not relevant